### ISSUE TITLE
Modal doesn't work in Chrome

### ISSUE THREAD

**Issue by c0brarules** [nico3333fr/jquery-accessible-modal-window-aria#5]
Hi Nico,

I hope you are able to translate this properly from English to French.

I appreciate you taking the time to create a simple and accessible modal window, using ARIA. This is something I feel can be very helpful to me.

Unfortunately I cannot seem to get any of the sample pages I have created, based on the code I have either taken from your Demo page here: http://a11y.nicolas-hoffmann.net/modal/ or your GitHub files here: https://github.com/nico3333fr/jquery-accessible-modal-window-aria, to reveal the pop-up modal in Google Chrome.

Here is a sample of the code I have been using http://codepen.io/c0brarules/pen/oXwdZQ based on your demo however the Modal only appears in Internet Explorer and Firefox but does not work in Google Chrome (I'm currently using Google Chrome Version 43.0.2357.124 m).

For some reason your actual Demo page works in Google Chrome though.

Do you know what I need to add to make mine to get it to appear?

Thanks.

- Dwayne

**Comment by nico3333fr** [comment]
Hi,

yes, no problem with english.

It is strange, as it works without problem on Chrome. I've update the sample, could you test ?

Kind regards,
Nicolas

**Comment by c0brarules** [comment]
Hi Nicolas,

Thanks for looking into this for me.

When you say it works without any problems on Chrome, are you referring to your Demo page here: http://a11y.nicolas-hoffmann.net/modal/ or your GitHub files here: https://github.com/nico3333fr/jquery-accessible-modal-window-aria, or the codepen http://codepen.io/c0brarules/pen/oXwdZQ sample I supplied? Which sample were you testing it with?

Which sample did you update for me to test? The one I included in codepen http://codepen.io/c0brarules/pen/oXwdZQ?

If so, it still only displays a transparent gray overlay but no modal box appears in Chrome Version 43.

Thanks again.
- Dwayne
